### DISCUSSION TITLE
Name collision

### DISCUSSION THREAD

**Discussion by wrossmann** [FerretDB/FerretDB#53] [General]
You might want to consider a rename.

https://github.com/dcramer/mangodb

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
ManosDB:
https://en.wikipedia.org/wiki/Manos:_The_Hands_of_Fate
¯\_(ツ)_/¯

**Comment by OpenSauce** [comment]
ThanosDB

**Comment by TheSYNcoder** [comment]
How about PosMonDB,
Taking good of both worlds :3

**Comment by ghost** [comment]
MongrelDB -- a mongrel is "an animal resulting in the crossing of different breeds or types". Mixing Mongo and Postgres counts I think?

**Comment by LordBurtz** [comment]
tbh change to MongrelDB seems good, the refactoring needed is minimal.. I really like thomascallahandell's idea

**Comment by docteurklein** [comment]
mongres

**Comment by CherryDT** [comment]
Well, with the other MangoDB being a joke project, I don't see a big issue here...

**Comment by coder1706** [comment]
MangroveDB - like mangrove forest ...

[a comment by an automated account is not shown]

**Comment by ogarcia** [comment]
MingaDB

**Comment by OpenSauce** [comment]
ModuloDB?

In computing, the modulo operation returns the remainder or signed remainder of a division, after one number is divided by another (called the modulus of the operation).
